packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/Microsoft.DotNet.Wpf.ProjectTemplates.ArchNeutral.csproj

```diff
@@ -24,6 +24,6 @@
 
   <ItemGroup>
     <PackagingContent Include="useSharedDesignerContext.txt" SubFolder="root" />
-    <PackagingContent Include="content\**\*" SubFolder="root\%(RecursiveDir)" />
+    <PackagingContent Include="content\**\*" SubFolder="root\content\%(RecursiveDir)" />
```

Comment:
Our current project template package looks like this: 

![image](https://user-images.githubusercontent.com/20246435/55822907-6b763480-5ab5-11e9-9233-d50686db5b66.png)

WinForms looks like this:

![image](https://user-images.githubusercontent.com/20246435/55822930-79c45080-5ab5-11e9-85fd-3785f0123ca3.png)

After your change, the hierarchy looks like this:

![image](https://user-images.githubusercontent.com/20246435/55823076-cd369e80-5ab5-11e9-8d9c-bd7d22f0e0f8.png)

- Is this an intentional change? 
- If so, what's the background? 
- What change is being made on the consumer side (dotnet/toolset?) to accomodate this change in schema, and who is responsible for coordinating it?

Reply:
Yes, because I have found that each other packages (e.g., `common.itemtemplates`, `common.projecttemplates`, `test`, `web`) has a `content` folder.
I have tested that the consumer side doesn't need to change.

Reply:
![image](https://user-images.githubusercontent.com/37586447/55843649-e6018b80-5b6b-11e9-8318-99cf9bc781c4.png)
The package in sdk preview3 release doesn't contains the `xlf` folder, is it by design?

Reply:
[Here](https://github.com/dotnet/templating/blob/stabilize/template_feed/Microsoft.DotNet.Common.ProjectTemplates.3.0/Microsoft.DotNet.Common.ProjectTemplates.3.0.csproj) is the project template sample, and it put all templates into `content` folder.

Reply:
Reviewer of the winforms repo [told](https://github.com/dotnet/winforms/pull/713#discussion_r274684762) me that it is not right, should I revert it?

Reply:
It seems you are busy, so I will revert it.😄